user_tasks/signals.py

```diff
@@ -75,15 +75,15 @@ def _create_chain_entry(user_id, task_id, task_class, args, kwargs, callbacks, p
             if parent_name and not parent.name:
                 parent.set_name(parent_name)
     for callback in callbacks:
+        LOGGER.debug(callback)
```

Comment:
@jmbowman I am trying to verify this block on sandbox. But not sure in which scenario this block will run.
I am importing/exporting course with olx validation and its adding entries in `usertaskstatus` table.